ckanext/switzerland/templates/scheming/form_snippets/temporals.html

```diff
@@ -29,5 +31,5 @@
       id="ogdch_temporals_add_row"
       class="btn btn-default ogdch-btn-add-row"
       href="#ogdch_temporals_pagemark">
-{{ _('Add another temporal') }}
+{{ _('Add row') }}
```

Comment:
Same as above. If we can already translate all those labels separetly, what is the reason to them to `Add row`?